coral/permissions/casbin.py

```diff
@@ -732,6 +732,13 @@ def get_restricted_instances(self, user, search_engine=None, allresources=False)
 
         if allresources is False and user.is_superuser is True:
             return []
+        
+        if allresources is True:
+            # Loop through casbin table for ids that match the user id and container resource instances
+            # -> Logical sets will have some sort of implementation here as well for gather rids
+            # Create array of rids the user has access to
```

Comment:
Is 744-754 not sufficient to cover this case, @taylorn01 ? (in particular, if we can avoid hitting the DB, as opposed to ES, then it'll be faster)

Reply:
Oh, I see - search_engine wasn't necessarily set, strange.